python-regression/tests/features/steps/api_test_steps.py

```diff
@@ -228,3 +232,63 @@ def check_node_sync(step, node, milestone):
         "Latest Solid Milestone {} on {} is not the expected {}".format(latestSolidMilestone, node, milestone)
 
 
+@step(r'A transaction is issued on "([^"]+)" with:')
+def issue_a_transaction(step, node):
+    """
+    Issues a single transaction
+
+    :param node: The identifier for the node (ie nodeA)
+    """
+    world.config['nodeId'] = node
+    api = api_utils.prepare_api_call(node)
+
+    transaction = transactions.evaluate_and_send(api, "", step.hashes)
+    transaction_hash = Transaction.from_tryte_string(transaction.get('trytes')[0]).hash
+    world.responses['transaction_hash'] = transaction_hash
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['world' may be undefined, or defined from star imports: aloe](https://app.codacy.com/app/GalRogozinski/iri/pullRequest?prid=4517274)